modules/core/exported/client.go

```diff
@@ -194,20 +194,12 @@ type ConsensusState interface {
 	ValidateBasic() error
 }
 
-// Misbehaviour defines counterparty misbehaviour for a specific consensus type
-type Misbehaviour interface {
+// ClientMessage is an interface used for Header and Misbehaviour implementations
```

Comment:
@AdityaSripal suggestion would be welcome for the comment/description here.

Reply:
Maybe something along the lines:
```
ClientMessage is an interface used to update an IBC client. The update may be done by a single header, a batch of headers, misbehaviour, or any type which when verified produces a change to state of the IBC client